### PR TITLE
Update: Tracked project page updated with next set of forked project details.

### PR DESCRIPTION
**Pull request by jobyko**

### CODE REVIEW

**Comment by jobyko** [comment]
@asa1997 
Please share your comments. 

@sumodgeorge

File: docs/projects_we_track.md [hunk 1]
```diff
@@ -568,3 +568,129 @@ The tool also checks for CVE issues and security advisories related to the CMS/f
...
+website with an API at asciinema.org,
+javascript player
+
+When you run asciinema rec in your terminal the recording starts, capturing all output that is being printed to your terminal while you’re issuing the shell commands. When the recording finishes (by hitting Ctrl-D or typing exit) then the captured output is uploaded to asciinema.org website and prepared for playback on the web.
+</p>
+<p align="right"><a href="https://github.com/be-secure/asciinema"> LEARN MORE >> </a></p></p>
+
+
+
+<h2><b>deploy</b></h2>
```
Comment (asa1997) [inline docs/projects_we_track.md:664]: @jobyko  I believe this talks about ansistrano/deploy & ansistrano/rollback, suggest you to change the headings accordingly.

**Review by asa1997 — COMMENTED** [review]
@jobyko Please address the comment.

**Comment by jobyko** [comment]
@asa1997 
Updated the change

**Comment by jobyko** [comment]
@panickervinod  @sumodgeorge 
Please merge the pull request.


@asa1997

**Review by asa1997 — APPROVED** [review]
@jobyko @panickervinod LGTM.

**Comment by jobyko** [comment]
@panickervinod @sumodgeorge 

Please check this PR and merge if it is fine.

**Comment by jobyko** [comment]
@panickervinod @sumodgeorge
Please merge the pull request if it fine. Already received LGTM for the same. 

@asa1997

**Pull request merged**